astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Accuracy tests for AltAz to ICRS coordinate transformations.
+
+We use "known good" examples computed with other coordinate libraries.
+
+Note that we use very low precision asserts because some people run tests on 32-bit
+machines and we want the tests to pass there.
+TODO: check if these tests pass on 32-bit machines and implement
+higher-precision checks on 64-bit machines.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from ....tests.helper import pytest, catch_warnings
+from .... import units as u
+from ....time import Time
+from ...builtin_frames import AltAz
+from ... import EarthLocation
+from ... import Angle, SkyCoord
+
+
+# Observatory position for `kpno` from here:
+# http://idlastro.gsfc.nasa.gov/ftp/pro/astro/observatory.pro
+kpno = EarthLocation(lon=Angle('111d36.0m'),
+                     lat=Angle('31d57.8m'),
+                     height=2120. * u.m)
+
+
+def test_against_hor2eq():
+    """Check that Astropy gives consistent results with an IDL hor2eq example.
+
+    See EXAMPLE input and output here:
+    http://idlastro.gsfc.nasa.gov/ftp/pro/astro/hor2eq.pro
+    """
+    obstime = Time('2041-12-25 22:00:00')
```

Comment:
In the example .pro file, it is 10 PM *local time*, while you enter a UT. It should be `'2041-12-26 06:00:00'`.